modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -36,9 +36,26 @@ Although token enforcement policies can optionally skip client validation, it is
 
 Before a client application is allowed to consume an API protected by a Client ID Enforcement policy, the client application must request access to the API. After an approved contract exists between the client application and the API, every request must include the client application credentials in compliance with how the policy is configured.
 
-For example, if the policy is configured to expect a client ID and client secret as headers, the application must send those credentials in the request, using the corresponding headers:
+For example, if the policy is configured to expect a client ID and client secret as query parameters, the application must send those credentials in the request. To enforce this, add a `client-id-required` RAML trait to the API specification:
 
-image::raml-snippet.png[]
+----
+traits:
+  client-id-required:
+    queryParameters:
+      client_id:
+        type: string
+      client_secret:
+        type: string
+----
+
+Apply the trait to the resource or methods using the `is` RAML attribute:
```

Comment:
confirm--resource is singular but methods is plural

Reply:
Fixed!